### PR TITLE
Todo list and readme 

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Hey folks please review my PR for README file and todo list feature.

Live Link: https://calm-syrniki-b1adc1.netlify.app/

focus

### CODE REVIEW

**Review by dev-prabhat — APPROVED** [review]
Looks good to me , you can merge this branch

File: src/pages/dashboard/dashboard.css [hunk 1]
```diff
@@ -191,13 +202,34 @@
...
+    display: flex;
+    flex-direction: row-reverse;
+    margin: 1rem 3rem;
+}
+
+.task-button{
+    position: relative;
+    background-color: transparent;
+    font-size: 1.6rem;
+    /* font-weight: 600; */
```
Comment (zoef-yt) [inline src/pages/dashboard/dashboard.css]: remove commented code.

File: src/pages/dashboard/dashboard.css [hunk 2]
```diff
@@ -169,7 +169,18 @@
     border-radius: 3px;
     background-color: var(--WHITE);
     z-index: 10;
-    overflow: hidden;
+    /* overflow: hidden; */
```
Comment (zoef-yt) [inline src/pages/dashboard/dashboard.css]: remove commented code.

**Review by humzahasan — APPROVED** [review]
UI looks great! Code changes looks good to merge.

**Review by zoef-yt — APPROVED** [review]
Good work, You are good to merge.

**Review by sanjitsarkar — APPROVED** [review]
- Code looks clean and readable
- Clean and Simple UI
- Functionality is working as expected
Good to merge

**Review by bossmonkey98 — APPROVED** [review]
Code looks clean good to merge

**Pull request merged**